riotfile.py

```diff
@@ -758,6 +758,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 "requests": [latest],
                 "redis": ">=2.10,<2.11",
                 "psycopg2-binary": [">=2.8.6"],  # We need <2.9.0 for Python 2.7, and >2.9.0 for 3.9+
+                "pandas": [latest],
```

Comment:
I'd avoid adding this potentially big dependency here if we could just test with a custom class that recreates the issue (e.g. the example provided by the issue reporter). WDYT?

Reply:
good point, will do